bindings/rust/extended/s2n-tls/src/connection.rs

```diff
@@ -1000,6 +1000,34 @@ impl Connection {
         }
     }
 
+    pub fn kem_group_name(&self) -> Option<&str> {
```

Comment:
Copy/Paste of `kem_name` function a few lines above, but with a call to `s2n_connection_get_kem_group_name()` instead of `s2n_connection_get_kem_name()`.